middleware/proxy_1_11_n.go

```diff
@@ -0,0 +1,13 @@
+// +build !go1.11
```

Comment:
Will it consider go > 1.11?

Reply:
Yes it will do. According to the documentation at: https://golang.org/pkg/go/build/#hdr-Build_Constraints

Excerpt:
```
[..]
- "go1.8", from Go version 1.8 onward
- "go1.9", from Go version 1.9 onward
- "go1.10", from Go version 1.10 onward
- "go1.11", from Go version 1.11 onward
[..]
```